dbms/src/Columns/ColumnNullable.cpp

```diff
@@ -454,13 +454,20 @@ void ColumnNullable::checkConsistency() const
 
 ColumnPtr makeNullable(const ColumnPtr & column)
 {
-    if (column->isColumnNullable())
+    if (checkColumn<ColumnNullable>(*column))
         return column;
 
-    if (column->isColumnConst())
+    if (isColumnConst(*column))
         return ColumnConst::create(makeNullable(static_cast<const ColumnConst &>(*column).getDataColumnPtr()), column->size());
 
     return ColumnNullable::create(column, ColumnUInt8::create(column->size(), 0));
 }
 
+bool isNullable(const IColumn & column)
```

Comment:
Pending LowCardinality.

Reply:
I've add IColumn.isNullble() and overridden it in ColumnConst, ColumnNullable and ColumnLowCardinality.